[2024-09-22T01:07:48.772+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_eval.group_4.DecisionTreeRegressor manual__2024-09-22T01:07:42.039789+00:00 [queued]>
[2024-09-22T01:07:48.816+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_eval.group_4.DecisionTreeRegressor manual__2024-09-22T01:07:42.039789+00:00 [queued]>
[2024-09-22T01:07:48.817+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-09-22T01:07:48.921+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): group_4.DecisionTreeRegressor> on 2024-09-22 01:07:42.039789+00:00
[2024-09-22T01:07:48.963+0000] {standard_task_runner.py:60} INFO - Started process 4812 to run task
[2024-09-22T01:07:48.976+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_eval', 'group_4.DecisionTreeRegressor', 'manual__2024-09-22T01:07:42.039789+00:00', '--job-id', '82', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmpmgk58v8t']
[2024-09-22T01:07:48.977+0000] {standard_task_runner.py:88} INFO - Job 82: Subtask group_4.DecisionTreeRegressor
[2024-09-22T01:07:49.023+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-09-22T01:07:49.201+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_eval.group_4.DecisionTreeRegressor manual__2024-09-22T01:07:42.039789+00:00 [running]> on host cc45b92b61e2
[2024-09-22T01:07:49.827+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_eval' AIRFLOW_CTX_TASK_ID='group_4.DecisionTreeRegressor' AIRFLOW_CTX_EXECUTION_DATE='2024-09-22T01:07:42.039789+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-22T01:07:42.039789+00:00'
[2024-09-22T01:07:50.001+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-09-22T01:07:50.055+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_eval, task_id=group_4.DecisionTreeRegressor, execution_date=20240922T010742, start_date=20240922T010748, end_date=20240922T010750
[2024-09-22T01:07:50.152+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-09-22T01:07:50.208+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
